playlist_souffle/entities/souffler.py

```diff
@@ -0,0 +1,29 @@
+"""Module for souffler entity."""
+
+from playlist_souffle.definitions.playlist import Playlist
+from playlist_souffle.entities.souffler_util import (
+    generate_souffle_description,
+    generate_souffle_name,
+    souffle_tracks
+)
+
+def generate_souffled_playlist(original_playlist, shuffle_by, collection_tracks_by_track, souffle_time):
```

Comment:
This should actually take in the user_id of the souffled playlist owner. Not necessarily the same as that of the original playlist.

This may also require a fix in the use case logic that may falsely assign the original playlist owner.